HacktoberfestProject.Web/wwwroot/css/site.css

```diff
@@ -12,6 +12,10 @@ a {
   color: #0366d6;
 }
 
+.error-txt {
```

Comment:
I thought there was a bootstrap class for this?

Reply:
`.text-danger` you are correct and I have changed.